uninit_filter.py

```diff
@@ -26,10 +29,25 @@ def find_uninit_classes(coverage_file):
     return uninit_dict
 
 
+def output(uninit_dict, output_file):
+    if output_file is not None:
+        with open(output_file, 'w') as out:
+            for file in uninit_dict:
+                out.write(file + str(uninit_dict[file]) + "\n")
+    else:
+        for file in uninit_dict:
+            print(file + str(uninit_dict[file]))
```

Comment:
You should use string formating here.

Either old school `'%s: %s' % (file_key, cls_name)`

or new school '{}: {}'.format(file_key, cls_name)`  <--new school offers more flexibility and power so I suggest using it but I don't know it well, I have been just using the old % way even in Python3.